cpp/src/arrow/adapters/arrow-rados-cls/cls_arrow.cc

```diff
@@ -78,23 +83,28 @@ class RandomAccessObject : public arrow::io::RandomAccessFile {
     return std::make_shared<arrow::Buffer>("");
   }
 
+  /// Read the byte stream. Returns a buffer of the objects in the stream.
```

Comment:
We can say "Read a specified number of bytes from the current position"

Reply:
Done.